services/OrderService.js

```diff
@@ -0,0 +1,11 @@
+import { OrderDao } from '../models';
+
+const findCartItems = async (id) => {
+  return await OrderDao.findCartItems(id);
+};
+
+const updateCartItemQuantity = async (id, cart_item_id, quantity) => {
+  return await OrderDao.updateCartItemQuantity(id, cart_item_id, quantity);
+};
```

Comment:
```suggestion

const findCartItems = async (userId) => {
  return await OrderDao.findCartItems(userId);
};

const updateCartItemQuantity = async (id, cart_item_id, quantity) => {
  return await OrderDao.updateCartItemQuantity(id, cart_item_id, quantity);
};
```
위에서 드렸던 리뷰와 같은 내용입니다 :)
나중에 장바구니에 있는 상품 하나하나를 조회하는 get 메소드가 생긴다면 
그때 함수 이름이
`findCartItem(id)`가 되면 좋겠네요 :)

Reply:
지적해주신 부분에 대해 위의 리뷰에서 함께 수정했습니다!

https://github.com/wecode-bootcamp-korea/fullstack1-1st-BaseNote-backend/pull/8#discussion_r670117642

소헌님 위의 리뷰의 `userId`처럼
장바구니에 있는 상품 하나하나를 조회하는 메소드의 경우에는
상품의 `id`라는 뜻으로 `productId`라고 지어주는 것은 어떤가요? 왜 `id`로 추천을 해주셨는지 궁금합니다!

Reply:
@iamhayoung 하영님 produtId도 좋습니다. 그런데 카트에 담긴 데이터 하나하나를 조회하는 함수는 사실 백엔드 입장에서 그건 진짜 product의 id가 아니라 cart를 테이블의 요소 하나이기 때문입니다 :) `cartItemId`도 괜찮겠네요 :) `productId`는 `products`테이블의 `id`로 혼동될 수 있습니다

Reply:
그렇군요! 앞으로 백엔드 입장에서의 데이터라는 것을 고려해서 이름을 부여해주겠습니다.
답변 감사합니다!